controllers/common.go

```diff
@@ -76,8 +87,75 @@ func BuildOpAMPBridge(params manifests.Params) ([]client.Object, error) {
 	return resources, nil
 }
 
+func (r *OpenTelemetryCollectorReconciler) findOtelOwnedObjects(ctx context.Context, params manifests.Params) (map[types.UID]client.Object, error) {
```

Comment:
nit: this method should be moved to the collector controller rather than be in common here IMO